nam/train/cli.py

```diff
@@ -225,11 +225,15 @@ def main_inner(
     model.net.export(outdir)
 
 
-if __name__ == "__main__":
+def main():
```

Comment:
[Nit] Now that there are 3 different "entry points" with different interfaces, it'd probably be good to docstring why they're all here and what they achieve [I can handle this.]